lib/src/license_detection/crc32.dart

```diff
@@ -0,0 +1,287 @@
+// Copyright (c) 2017, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// Specification of [CRC-32][1] is same as used in IEEE 802.3 Ethernet (Normal).
```

Comment:
The `crc32` is a function... this doesn't parse:
```dart
/// Specification of [CRC-32][1] is same as used in IEEE 802.3 Ethernet (Normal).
```

You probably want to write something along the lines of.
```dart
/// Computes CRC-32 of [bytes].
///
/// This follows the [CRC-32][1] specification used in IEEE 802.3 Ethernet (Normal).
```